modules/Bio/EnsEMBL/IO/Parser/Genbank.pm

```diff
@@ -92,13 +92,16 @@ sub read_record {
             $self->{'record'}->{'_raw_accession'} = $field;
         }
         elsif ($field_type eq 'VERSION') {
-            if ($field =~ /\S+\.(\d+)\s+GI:(\d+)/) {
+            if ($field =~ /\S+\.(\d+)/) {
                 $self->{'record'}->{'_version'} = $1;
                 $self->{'record'}->{'_genebank_id'} = $2;
```

Comment:
There is no capture for $2 in ($field =~ /\S+\.(\d+)/).  Think _genebank_id will never be initialised there?!

Reply:
You are correct this line should have been removed